streams/examples/src/main/java/org/apache/kafka/streams/examples/wordcount/WordCountDemo.java

```diff
@@ -62,7 +61,7 @@ public static void main(String[] args) throws Exception {
 
         StreamsBuilder builder = new StreamsBuilder();
 
-        KStream<String, String> source = builder.stream("streams-wordcount-input");
+        KStream<String, String> source = builder.stream("streams-wordcount-input", "dummy");
```

Comment:
What is `dummy` ?

Reply:
As mentioned in the PR:

```
Changes in WordCountDemo.java are intentional for illustration purposes, and I will revert the changes after addressing review comments.
```

Ditto below.